homework/Danila/Animals/src/by/jrr/thread/RunMyThread.java

```diff
@@ -0,0 +1,12 @@
+package by.jrr.thread;
+
+import by.jrr.children.Cat;
+import by.jrr.parents.Animal;
+
+public class RunMyThread implements Runnable {
+    @Override
+    public void run() {

```

Comment:
add all animals to collection and use common methods of a superclass and interface